src/proxy.c

```diff
@@ -689,10 +683,55 @@ int parse_json_from_stdin (const char *section) {
 
 void print_usage()
 {
-    g_print("dbus-proxy, version %s\n", PACKAGE_VERSION);
-    g_print("Usage: dbus-proxy address session|system\n"
-            "waits for JSON conf at stdin.\n");
+    g_message("dbus-proxy, version %s\n", PACKAGE_VERSION);
+    g_message("Usage: dbus-proxy address session|system\n"
+              "waits for JSON conf at stdin.\n");
```

Comment:
Do we really want to use the log handler for this output? Shouldn't this always go to stdout?